samples/customHandler/jsonRpc.js

```diff
@@ -0,0 +1,18 @@
+export function wrap(env) {
+  return async (json) => {
+    const request = new Request("http://fake-host", { method: "POST", body: JSON.stringify(json) } );
+    return await (await env.target.fetch(request)).json();
+  }
+}
+
+export function unwrap(target) {
+  const { fetchJson, ...rest } = target;
+  return {
+    ...rest,
+    async fetch(req, env) {
```

Comment:
It seems pretty rough that this uses up the `fetch` handler, meaning that the worker that wants to export a `fetch` handler can't also implement its own `fetch` handler. For example, it's quite normal for queues users to define both a `fetch` handler (that accepts incoming HTTP requests and pushes messages into a queue, along with whatever other work is needed) and a `queue` handler (that handles those messages) in the same script. But in this model that wouldn't be possible.

Should we be doing something here along the lines of checking a special header on the request to determine if the request is meant for a custom handler, and if not then forward it to `target`'s existing `fetch` handler (if it has one)?